config/_templates/dataset/yaak/predict.yaml

```diff
@@ -340,4 +341,5 @@ samples:
                     list_max("meta/VehicleMotion/brake_pedal_normalized") <= (1.0 / 164 + 0.001) AND
                     list_max("meta/VehicleMotion/speed") >= 25.0 AND
                     list_last("meta/VehicleMotion/speed") - list_first("meta/VehicleMotion/speed") >= -0.05 * list_avg("meta/VehicleMotion/speed")
-                )
\ No newline at end of file
+                )
+                AND list_max(apply(FLATTEN(FLATTEN("waypoints/waypoints_normalized")), x -> ABS(x))) <= 300
```

Comment:
just a thought: the filter above ("no cruise control") needs to work on the entire window (t=24 in this case), while this one doesn't IIUC (please correct me if i'm wrong) -- would it make sense to filter outlier waypoints earlier upstream? ie right after they're produced by `WaypointNormalizer`